local-running/setConfigForLocalOrCloudRunning.py

```diff
@@ -239,34 +239,49 @@ def determine_dev_account():
         print(f"Looks like you're not authenticated against a dev account ({account_id}). Cowardly refusing to write params.")
     return DevAccount(account_id, account_num)
 
-def set_async_lambda_event_source_mapping_enabled_state(environment, account_id, dry_run, enabled):
+def delete_async_lambda_event_source_mapping(environment, account_id, dry_run):
     get_event_mapping_uuid_result = subprocess.run(
         ['aws', 'lambda', 'list-event-source-mappings', '--function-name', f'arn:aws:lambda:eu-west-2:{account_id}:function:process-async-cri-credential-{environment}:live', '--query', "EventSourceMappings[0].UUID", '--output', 'text'],
         capture_output=True,
         text=True
     )
 
     if get_event_mapping_uuid_result.returncode != 0:
-        print(f"Could not get event mapping UUID. Cowardly refusing to continue.")
+        print("Could not get event mapping UUID. Cowardly refusing to continue")
         exit(1)
 
     event_mapping_uuid = get_event_mapping_uuid_result.stdout.strip()
     print(f"Found event mapping with UUID: {event_mapping_uuid}")
 
-    enabled_flag = "--enabled" if enabled else "--no-enabled"
     if not dry_run:
-        update_event_mapping_return_code = subprocess.run(
-            ['aws', 'lambda', 'update-event-source-mapping', '--uuid', f'{event_mapping_uuid}', enabled_flag],
+        delete_event_mapping_return_code = subprocess.run(
+            ['aws', 'lambda', 'delete-event-source-mapping', '--uuid', f'{event_mapping_uuid}'],
             stdout=subprocess.DEVNULL,
             stderr=subprocess.DEVNULL
         ).returncode
 
-        if update_event_mapping_return_code != 0:
-            print(f"Could not update event mapping enabled state. Cowardly refusing to do anything else.")
+        if delete_event_mapping_return_code != 0:
+            print("Could not delete event mapping. Cowardly refusing to do anything else")
             exit(1)
-        print(f"process-async-cri-credential lambda event mapping state set to: {enabled_flag}")
+        print("process-async-cri-credential lambda event mapping deleted")
     else:
-        print(f"process-async-cri-credential lambda event mapping state would have been set to: {enabled_flag}")
+        print("process-async-cri-credential lambda event mapping would have been deleted")
+
+def create_async_lambda_event_source_mapping(environment, account_id, dry_run):
+    create_event_mapping_uuid_return_code = subprocess.run(
```

Comment:
Should this be inside the `if not dry_run`? It looks like it actually does the create...

Reply:
Whoops